### ISSUE TITLE
Notification already showing, but can't link ERROR aiohttp.server

### ISSUE THREAD

**Issue by serialtasted** [marcelveldt/hass_emulated_hue#11]
I'm having some trouble where I can see in the logs the app requesting to connect but no notification to enable the link.

emulated_hue version: 0.0.4
hassio version: 0.109.6
supervisor version: 222

**Comment by serialtasted** [comment]
This solved by itself... Weird things. But now I'm getting another error when trying to pair:
```
2020-05-11 18:06:41,990 ERROR aiohttp.server -- Error handling request
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/aiohttp/web_protocol.py", line 418, in start
    resp = await task
  File "/usr/lib/python3.7/site-packages/aiohttp/web_app.py", line 458, in _handle
    resp = await handler(request)
  File "/usr/local/app/emulated_hue/hue_api.py", line 71, in func_wrapper
    return await func(cls, request)
  File "/usr/local/app/emulated_hue/hue_api.py", line 179, in get_lights
    return web.json_response(await self.__get_all_lights())
  File "/usr/local/app/emulated_hue/hue_api.py", line 691, in __get_all_lights
    result[light_id] = await self.__entity_to_json(entity)
  File "/usr/local/app/emulated_hue/hue_api.py", line 665, in __entity_to_json
    model_slug = slugify(retval["modelid"]).upper()
  File "/usr/local/app/emulated_hue/utils.py", line 50, in slugify
    return unicode_slug.slugify(text, separator="_")  # type: ignore
  File "/usr/lib/python3.7/site-packages/slugify/slugify.py", line 104, in slugify
    text = _unicode(text, 'utf-8', 'ignore')
TypeError: decoding to str: need a bytes-like object, NoneType found
```

**Comment by mcl22** [comment]
I am currently running into the same error. I had it working several days ago. Then I uninstalled the addon and wanted to try it again yesterday. So I re-installed it and now I run into this problem. I already rebuilt and re-installed it. I also stopped, cleared cache, cleared data and re-installed the hue-app on my phone. Still no change.

When I tried it yesterday, the app found the bridge, the message to ha was sent and after enabling the link mode the app told me that there wasn't any new bridge. Now trying today the app doesn't find a new bridge at all but the message is still received in ha.

Any suggestions?
